lib/active_admin/batch_actions/controller.rb

```diff
@@ -36,6 +38,16 @@ def current_batch_action
       def batch_action_collection(only = COLLECTION_APPLIES)
         find_collection(only: only)
       end
+
+      def pick_valid_keys
+        inputs     = JSON.parse params[:batch_action_inputs]  || '{}'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.